### DISCUSSION TITLE
How to setup pre-dev commands

### DISCUSSION THREAD

**Discussion by ricardo-rp** [pnpm/pnpm#5758] [Q&A] [answered]
We're using a monorepo, and our frontend depends on other packages. When there are changes in the `shared` pacakges, we need to re-run `pnpm build` for the package.
Often, frontend devs will fire up the repo, attempt to open the dev server and find they needed to build the shared packages again because of new changes.
I know yarn has 'predev' and 'prebuild' commands.
Is there a similar feature in pnpm, or should I just concatenate my build commands before dev using && ?

**Comment by lvqq** [comment, marked as answer]
`predev` and `prebuild` scripts are supported, you need to add `enable-pre-post-scripts = true` option in `.npmrc` to enable it
